Tests/AccountTests/Push/AutopushTests.swift

```diff
@@ -0,0 +1,99 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import XCTest
+import MozillaAppServices
+import Account
+
+class AutopushTests: XCTestCase {
+    private var mockPushManager = MockPushManager()
+    private var autopushClient: Autopush?
+
+    override func setUp() async throws {
+        autopushClient = Autopush(withPushManager: mockPushManager)
+        try await super.setUp()
+    }
+
+    override func tearDown() {
+        autopushClient = nil
+        mockPushManager = MockPushManager()
+        super.tearDown()
+    }
+    func testSubscribeCallsPushManager() async throws {
```

Comment:
nit, spacing between funcs
```suggestion
    }

    func testSubscribeCallsPushManager() async throws {
```